Blitz/Crossfire/map.json

```diff
@@ -48,7 +48,7 @@
 		}
 	],
 	"itemremove": [
-		"string", "sponge", "arrow", "stick", "oak planks", "sign"
+		"leather helmet", "leather chestplate", "leather leggings", "string", "sponge", "arrow", "stick", "oak planks", "sign"

```

Comment:
"sign" isn't a valid item name, there's multiple types of signs like "oak sign" etc.